js/keyboard/button.js

```diff
@@ -0,0 +1,63 @@
+import {
+	WIDTH,
+	LANGUAGE,
+} from '../constants.js';
+
+export default class Button {
+	constructor(opts) {
+		const {
+			code,
+			enValue,
+			ruValue,
+			enValueAdd,
+			ruValueAdd,
+			width,
+			language,
+		} = opts || {};
+		this.code = code;
+		this.isRegular = enValue.length === 1;
+		this.enValue = enValue.length > 1 ? enValue : enValue.toLowerCase();
+		this.ruValue = ruValue.length > 1 ? ruValue : ruValue.toLowerCase();
+		this.enValueAdd = enValueAdd || (enValue.length > 1 ? enValue : enValue.toUpperCase());
+		this.ruValueAdd = ruValueAdd || (ruValue.length > 1 ? ruValue : ruValue.toUpperCase());
+		this.width = width || WIDTH.S;
+		this.language = language || LANGUAGE.EN;
+		this.el = document.createElement('div');
```

Comment:
It will be better to use `this.el` only in this class but not in the `Keyboard`. All the methods and 'addEventListeners' that concern this variable should be in this class. That way you could avoid a mess between classes.